packages/api/test/e2e/cennzx.e2e.ts

```diff
@@ -107,25 +105,28 @@ describe('CENNZX RPC calls testing', () => {
           });
         });
 
-        // it('Query estimated fee in different currency (CENNZ)', async done => {
-        //   const maxPayment = '50000000000000000';
-        //   const transactionPayment = generateTransactionPayment({tip:0, assetId:CENNZ, maxPayment});
-        //   const extrinsic = api.tx.genericAsset
-        //     .transfer(PLUG, bob.address, 10000);
-        //   const feeFromQuery = await api.derive.fees.estimateFee({extrinsic, userFeeAssetId: CENNZ, maxPayment});
-        //   await extrinsic.signAndSend(alice,  {transactionPayment}, async ({events, status}) => {
-        //     if (status.isFinalized) {
-        //       events.forEach(({phase, event: {data, method, section}}) => {
-        //         if (method === 'AssetPurchase') {
-        //           const price = data[3];
-        //           expect(feeFromQuery).toEqual(price);
-        //         }
-        //         console.log('\t', phase.toString(), `: ${section}.${method}`, data.toString());
-        //       });
-        //       done();
-        //     }
-        //   });
-        // });
+        it('Query estimated fee in different currency (CENNZ)', async done => {
+          const maxPayment = '50000000000000000';
+          const assetId = api.registry.createType('AssetId', CENNZ);
+          const feeExchange = api.registry.createType('FeeExchange', {assetId, maxPayment}, 0);
+          const transactionPayment = api.registry.createType('ChargeTransactionPayment', {tip: 0, feeExchange});
+          const extrinsic = api.tx.treasury.reportAwesome('Fantastic Work', alice.address);
```

Comment:
Used different extrinsic to check if estimate fee in CENNZ works fine.. earlier there was a transfer extrinsic using PLUG